magenta/models/polyphonic_rnn/polyphonic_rnn_lib.py

```diff
@@ -199,14 +199,17 @@ def duration_and_pitch_to_midi(filename, durations, pitches, prime_until=0):
   sn = 4
 
   # Translate durations from TIME_CLASSES indexes to TIME_CLASSES values.
+  # First, find the data to change, then change it.
+  # If we don't do this as separate steps, we'll end up changing some data
+  # more than once.
   dt = copy.deepcopy(durations)
-  for i, time_class in enumerate(TIME_CLASSES):
-    dt[dt == i] = time_class
+  time_class_indexes = []
```

Comment:
indexes is more common in North America: http://grammarist.com/usage/indexes-indices/